super_project/CMakeLists.txt

```diff
@@ -16,17 +12,34 @@ else()
 endif()
 
 
-list(APPEND CMAKE_MODULE_PATH
-    ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-
+include(FetchContent)
+FetchContent_Declare(
+  farm_ng_cmake
+  GIT_REPOSITORY https://github.com/farm-ng/farm-ng-cmake.git
+  GIT_TAG main
+)
+FetchContent_MakeAvailable(farm_ng_cmake)
+set(farm_ng_cmake_DIR ${farm_ng_cmake_SOURCE_DIR}/cmake)
 set(BUILD_SHARED_LIBS ON)
-include(farm_ng_default_args)
+find_package(farm_ng_cmake REQUIRED)
 
 include(External_fmt)
 include(External_expected)
 include(External_eigen)
 include(External_ceres)
-include(External_farm-ng-core)
+
+ExternalProject_Add(farm-ng-core
+    DEPENDS fmt expected eigen
+    GIT_REPOSITORY  "https://github.com/farm-ng/farm-ng-core.git"
+    GIT_TAG "main"
```

Comment:
@ethanrublee that makes the build non-deterministic. I'd prefer to track a specific sha or release tag.

Reply:
i made a v0.1.0 release.